rest-service/manager_rest/rest/resources_v3_1/deployments.py

```diff
@@ -324,3 +333,82 @@ def get(self,
                 substr_filters=search
             )
         return inter_deployment_dependencies
+
+
+class InterDeploymentDependenciesRestore(SecuredResource):
+    @authorize('inter_deployment_dependency_create')
+    def post(self):
+        """
+        Updating the inter deployment dependencies table from the specified
+        deployment during an upgrade
+
+        """
+        data = self._get_request_data()
+        deployment_id = data.get('deployment_id')
+        runtime_only_evaluation = data.get('runtime_only_evaluation')
+        sm = get_storage_manager()
+        deployment = sm.get(models.Deployment, deployment_id)
+        blueprint = deployment.blueprint
+        app_dir = join(FILE_SERVER_BLUEPRINTS_FOLDER,
+                       utils.current_tenant.name,
+                       blueprint.id)
+        app_blueprint = blueprint.main_file_name
+        parsed_deployment = rest_utils.get_parsed_deployment(
+            blueprint, app_dir, app_blueprint)
+        deployment_plan = rest_utils.get_deployment_plan(
+            parsed_deployment, deployment.inputs,  runtime_only_evaluation)
+        rest_utils.update_deployment_dependencies_from_plan(
+            deployment_id, deployment_plan, sm, lambda *_: True)
+        if data.get('update_service_composition'):
+            self._create_service_composition_dependencies(deployment_plan,
+                                                          deployment,
+                                                          sm)
```

Comment:
doesn't this function need to return something?

Reply:
Great question, it basically just updates the inter deployment dependencies in the DB, so I figured it doesn't need to return anything.